### ISSUE TITLE
Where is HANLayer?

### ISSUE THREAD

**Issue by lxytypk** [catly/CE-DTI#1]
cmodel.py中的CE_DTI是没有写完整HANLayer吗？

**Comment by GuanyuQiao98** [comment]
HANLayer 改成HGenerativeGraphFusion

**Comment by haha521111** [comment]
> HANLayer 改成HGenerativeGraphFusion

你好，我已复现出代码使用的是github给出的参数--learning_rate=0.001 --dataname=zheng --dropout=0.2 --inp_size=128，但是为什么在zheng数据集上只能得到ROC：0.90，PR：0.89，heter数据集上ROC：0.94，PR：0.93，和论文中提到的0.97差不少呢，是我参数没调正确吗？感谢回答

**Comment by GuanyuQiao98** [comment]
1.检查Python以及依赖三方库版本是否与提供版本相同
不同版本在底层数值计算（如浮点精度、随机数种子初始化方式、依赖库实现细节）上可能存在微小差异。这种差异在可能在模型较深时逐步放大，最终影响模型性能指标，检查python环境是否与提供环境一致。
 2.服务器硬件环境带来的影响
不同的服务器架构也会对 GPU 上数值计算过程带来微小影响。同一模型在不同服务器上训练时，可能因浮点数误差或数据加载顺序不同而导致权重收敛略有偏差。

我们在两台不同的NVIDIA A100 80G服务器上配置相同环境并使用github提供代码进行了重复实验。模型性能指标仅存在千分位级别的偏差（如heter数据集ROC为0.975±0.002），表明训练过程在相同服务器架构以及总体是稳定且可重复的。

<img width="836" alt="Image" src="https://github.com/user-attachments/assets/99e4654f-9ca3-4570-bb8a-b5a589a743de" />

**Comment by haha521111** [comment]
你好，我想请问问什么您的实验的训练集验证集测试集都是划分好的？而不采用五折交叉验证呢？

**Comment by haha521111** [comment]
> > 你好，我想请问问什么您的实验的训练集验证集测试集都是划分好的？而不采用五折交叉验证呢？
> 
> > > HANLayer 改成HGenerativeGraphFusion
> > 
> > 
> > 你好，我已复现出代码使用的是github给出的参数--learning_rate=0.001 --dataname=zheng --dropout=0.2 --inp_size=128，但是为什么在zheng数据集上只能得到ROC：0.90，PR：0.89，heter数据集上ROC：0.94，PR：0.93，和论文中提到的0.97差不少呢，是我参数没调正确吗？感谢回答
> 
> 请问你解决了这个问题吗？我跑的也是和你一样的结果，test 0.8873056994818653 0.9493107197508658 0.9360873018558019

按照他给的参数就是这个结果，调了调参好了点，但还是不如论文中的结果

**Comment by jacks-ai** [comment]
> > > 你好，我想请问问什么您的实验的训练集验证集测试集都是划分好的？而不采用五折交叉验证呢？
> > 
> > 
> > > > HANLayer 改成HGenerativeGraphFusion
> > > 
> > > 
> > > 你好，我已复现出代码使用的是github给出的参数--learning_rate=0.001 --dataname=zheng --dropout=0.2 --inp_size=128，但是为什么在zheng数据集上只能得到ROC：0.90，PR：0.89，heter数据集上ROC：0.94，PR：0.93，和论文中提到的0.97差不少呢，是我参数没调正确吗？感谢回答
> > 
> > 
> > 请问你解决了这个问题吗？我跑的也是和你一样的结果，test 0.8873056994818653 0.9493107197508658 0.9360873018558019
> 
> 按照他给的参数就是这个结果，调了调参好了点，但还是不如论文中的结果

确实，我也调了，但是在zheng数据集上封顶也就 0.928  0.925

**Comment by catly** [comment]
感谢提醒，我们确认了上述中相关问题。
inp_size 的正确配置应为 256，之前的 128 属于配置错误。
相关参数已在最新提交中修复，请同步最新版本后重新尝试。
